components/textbox/textbox.jsx

```diff
@@ -245,6 +245,7 @@ export default class Textbox extends React.Component {
                     disabled={this.props.disabled}
                     contextId={this.props.channelId}
                     listenForMentionKeyClick={this.props.listenForMentionKeyClick}
+                    wrapperRef={this.refs.wrapper}
```

Comment:
do we need the full wrapper? if we are only using its height, why not passing just that like `wrapperHeight` (or some better name if you like)
one benefit is that there is less stuff moving around, but also you can check the prop for being a number, which is a lot more restrictive than a generic object.